test/sass4clj/core_test.clj

```diff
@@ -3,29 +3,8 @@
             [sass4clj.core :refer :all])
   (:import [java.io File]))
 
-(def sass
```

Comment:
The tests were passing, but I didn't understand what use case they were meant to represent, and they relied on the code I removed from `core.clj`.

It also seemed like they created ambiguity that runs counter to how Sass is used. `@import` is supposed to take a relative path that resolves to a single absolute path, but these tests were treating the import paths as relative to both the temporary file and the resources directory.

I'm sure I'm not understanding it correctly, but included the change in order to get your thoughts. I couldn't think of a case where you'd want a a sass file in a completely random location to be able to `@import` files relative to resources, and not just `@import` relative to its own location, which seems to be the standard behavior 😄 

Thanks for having a look! This library is very useful and well done.